.gitignore

```diff
@@ -66,3 +66,5 @@ ipch/
 CMakeLists.txt
 compile_commands.json
 /.gitmodules
+cscope.out
+EndlessSky.cscope_file_list
```

Comment:
```suggestion
```
I don't think you should be changing this here, its completely unrelated to the PR, if you want these changes for personal use you can just open a new PR for it, which will get merged super fast probably, if people agree.